plugin/vcs/github/github.go

```diff
@@ -315,6 +317,11 @@ func (p *Provider) FetchCommitByID(ctx context.Context, oauthCtx common.OauthCon
 	}, nil
 }
 
+// GetDiffFileList gets the diff files list between two commits.
+func (*Provider) GetDiffFileList(_ context.Context, _ common.OauthContext, _, _, _, _ string) ([]vcs.FileDiff, error) {
+	return nil, nil
```

Comment:
Are we going to implement this? If I understand the semantics correctly if we return an empty list that would mean always no files changed in the diff (that we should care about, which sounds the opposite of "not yet implemented")?

Reply:
Good catch. There's a hack here https://github.com/bytebase/bytebase/blob/19d4ffcd56716fee89e8945e8ec8f38a9cd52a50/server/webhook.go#L578.

I'll implement the GitHub version of GetDiffFileList and remove this hack.

We'll also use the diff as the source of file changes instead of a filter.